packages/lab/src/calendar/internal/CalendarDay.tsx

```diff
@@ -36,55 +34,57 @@ export const CalendarDay = forwardRef<HTMLButtonElement, CalendarDayProps>(
     });
 
     const dayRef = useRef<HTMLButtonElement>(null);
-    const { status, dayProps, unselectableReason } = useCalendarDay(
-      {
-        date: day,
-        month,
-      },
-      dayRef
-    );
-    const { outOfRange, today, unselectable, hidden } = status;
+    const buttonRef = useForkRef(ref, dayRef);
+    const { status, dayProps, unselectableReason, highlightedReason } =
+      useCalendarDay(
+        {
+          date: day,
+          month,
+        },
+        dayRef
+      );
+    const { outOfRange, today, unselectable, highlighted, hidden, disabled } =
+      status;
 
     return (
       <Tooltip
         hideIcon
-        status="error"
-        content={unselectableReason}
-        disabled={!unselectableReason}
+        status={unselectableReason ? "error" : "info"}
+        content={
+          unselectableReason || highlightedReason || "Date is out of range"
+        }
+        disabled={!unselectableReason && !highlightedReason}
         placement="top"
-        enterDelay={300}
+        enterDelay={0} // --salt-duration-instant
         {...TooltipProps}
       >
         <button
           aria-label={formatDate(day)}
           {...dayProps}
-          ref={dayRef}
+          ref={buttonRef}
           {...rest}
           className={clsx(
             withBaseName(),
             {
               [withBaseName("hidden")]: hidden,
               [withBaseName("outOfRange")]: outOfRange,
-              [withBaseName("today")]: today,
+              [withBaseName("disabled")]: disabled,
```

Comment:
Disabled days are not communicated to screen readers

Reply:
that is right, they should be unselectable, unfocusable and have no tooltip

Reply:
The disabled attribute isn't applied